telegram/ext/_handlers/chatmemberhandler.py

```diff
@@ -83,11 +95,13 @@ def __init__(
         self,
         callback: HandlerCallback[Update, CCT, RT],
         chat_member_types: int = MY_CHAT_MEMBER,
+        chat_id: Optional[SCT[int]] = None,
```

Comment:
please move this to the end of the parameter list - otherwise `CMH(callback, MY_CHAT_MEMBER, True)` will break